extras/jsonrpc/daemon.go

```diff
@@ -595,3 +601,42 @@ func (d *Client) AccountAdd(accountName string, seed *string, privateKey *string
 	structs.DefaultTagName = "json"
 	return response, d.call(response, "account_add", structs.Map(args))
 }
+
+type WalletCreateOpts struct {
+	SkipOnStartup bool
+	CreateAccount bool
+	SingleKey     bool
+}
+
+func (d *Client) WalletCreate(id string, opts *WalletCreateOpts) (*Wallet, error) {
+	response := new(Wallet)
+	if opts == nil {
+		opts = &WalletCreateOpts{}
+	}
+	params := map[string]interface{}{
+		"wallet_id":       id,
+		"skip_on_startup": opts.SkipOnStartup,
```

Comment:
you gotta remember to define the `json` part and add `,omitempty` otherwise the SDK will crap out

Reply:
I changed the way options are provided, have a look please